tests/test_meshtastic_utils.py

```
@@ -576,7 +576,50 @@ def test_connect_meshtastic_tcp(self, mock_tcp, mock_ble, mock_serial):
-        mock_tcp.assert_called_once_with(hostname="192.168.1.100", timeout=300)
+        mock_tcp.assert_called_once_with(
+            hostname="192.168.1.100",
+            portNumber=4403,
+            timeout=300,
+        )
+
+    @patch("mmrelay.meshtastic_utils.meshtastic.serial_interface.SerialInterface")
+    @patch("mmrelay.meshtastic_utils.meshtastic.ble_interface.BLEInterface")
+    @patch("mmrelay.meshtastic_utils.meshtastic.tcp_interface.TCPInterface")
+    def test_connect_meshtastic_tcp_uses_configured_port(
+        self, mock_tcp, mock_ble, mock_serial
```

Ruff (0.15.5)

[warning] Unused method argument: `mock_ble`

(ARG002)